### DISCUSSION TITLE
Netherite Plus

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#157] [Ideas]
I think Netherite Ingot can be used to do more things（Something like Netherite Shulker Boxes）

**Comment by Omgise** [comment]
There are more tools to expand.   :D

**Comment by Roadhog360** [comment]
I may integrate NetheritePlus+ features and add Netherite shulker boxes.

**Comment by Omgise** [comment]
Add Netherite support to the Aether mod.

**Comment by GregoriusT** [comment]
Wait what? What exactly would you use Netherite in Aether for?

**Reply by Omgise** [reply]
Allow diamond gear to be upgraded to netherite equipment.
